generate.js

```diff
@@ -14,10 +14,11 @@ var base = require("xbase"),
 	tiptoe = require("tiptoe");
 
 var HOTS_PATH = process.argv[2] || "/Applications/Heroes\ of\ the\ Storm";
+var HOTS_LANG = process.argv[3] || "enus";
 
 if(!fs.existsSync(HOTS_PATH))
 {
-	base.error("Usage: node generate.js /path/to/hots");
+	base.error("Usage: node generate.js /path/to/hots language");
```

Comment:
Please add brackets around `/path/to/hots` and `language` indicating they are optional.